### PR TITLE
SKY-3: Footer.html design2

### PR DESCRIPTION
**Pull request by anurmamedov**
uploaded footer.
Screenshot 2023-01-23 203653
Screenshot 2023-01-23 213344-medium-sixe
Screenshot 2023-01-23 213427- small

### CODE REVIEW

**Comment by anurmamedov** [comment]
Updated, review it.

**Comment by Aymammet** [comment]
in commits description try to write exactly what changes you made.
Example : "design of social media icons changed", "display style of footer columns changed so that they are not broken in small size screens.", "minor bugs fixed", "error in line 3 that was made in previous commit fixed, 'sm' is replaced with 'md' "

**Comment by Aymammet** [comment]
Send image of your changes. It is hard to see what changes you made by just looking at code. Especially in design changes. Place different screen size images, to get enough information how they look like.

**Comment by anurmamedov** [comment]
Now.
Screenshot 2023-01-23 203653

Before.
before-Screenshot 2023-01-23 203851

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+  <!-- Remove the container if you want to extend the Footer to full width.-->
```
Comment (Aymammet) [inline templates/footer.html:1]: Remove commented lines

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    </div>
+                </section>
+            </div>
+            <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2)">
+                 © 2023 Copyright: 
+                <a class="text-white " href="#">SKY-Social-Media</a>
+            </div>
+         </footer>
+    </div>
+</html>
```
Comment (Aymammet) [inline templates/footer.html:47]:
```suggestion
```

File: templates/footer.html (hunk 1 above)
Comment (Aymammet) [inline templates/footer.html:1]:
```suggestion
```

**Review by Aymammet — CHANGES_REQUESTED** [review]

**Review by Aymammet — APPROVED** [review]

**Pull request merged**